user-guide/Standard_Apps/IDP/Reference/Using_your_IS04_registry_to_provision_DataMiner_with_IDP.md

```diff
@@ -0,0 +1,172 @@
+---
+uid: Using_your_IS04_registry_to_provision_DataMiner_with_IDP
+---
+
+# Using your IS-04 registry to provision DataMiner with IDP
+
+Below, you can find how you can configure DataMiner IDP to use an NMOS IS-04 registry, so that the registered IS-04 nodes are provisioned in the DMS.
+
+## Installation
+
+- Make sure your DataMiner System has been upgraded to the correct minimum version required for the version of Process Automation that you intend to use, or preferably, simply upgrade to the [latest DataMiner feature release](https://community.dataminer.services/downloads/).
+
+- Install [DataMiner IDP with Process Automation](xref:Installing_DataMiner_IDP).
+
+- Make sure a DataMiner element is available that uses the connector [AMWA NMOS IS-04 Registry](https://catalog.dataminer.services/result/driver/6923). During element creation, the *IP address/host* of the connection *HTTP Connection Main* must be set to the IP address of the IS-04 registry.
+
+## Configuration
+
+### Adding the element property IS-04 Node GUID
+
+Add the custom element property *IS-04 Node GUID*, as described in [Adding a custom property to an item](xref:Managing_element_properties#adding-a-custom-property-to-an-item). Make sure to specify the options as shown in figure 1 below.
+
+![Configuration of element property IS-04 Node GUID](~/user-guide/images/IDP_IS-04_1.png)<br>
+*Figure 1: Configuration of element property IS-04 Node GUID*
+
+When an element is provisioned for an IS-04 node that registered itself to the IS-04 registry, the custom update property script will fill in this element property with the node's [Node ID](https://specs.amwa.tv/is-04/releases/v1.3.1/docs/5.1._Data_Model_-_Identifier_Mapping.html#node-id).
+
+It is important that the Node ID is stored on the provisioned elements, as this is how the connector *AMWA NMOS IS04 Registry* will identify if a node has already been provisioned. If this property is not configured or not filled in by the custom update property script, each node update will trigger the provisioning of a new element.
+
+> [!NOTE]
```

Comment:
Instead of having this as a note, please put this as first paragraph because IDP 1.1.15 is from Aug 2021 and I expect most users to have this version (or higher) already. 
The property *IS-04 Node GUID* has to be added via the CI Type Management wizard